cmd/migrate-db/migrateDB.go

```diff
@@ -0,0 +1,35 @@
+package migratedb
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/forbole/bdjuno/v2/database"
+	"github.com/forbole/juno/v2/cmd/parse"
+	"github.com/spf13/cobra"
+)
+
+// NewMigrateDBCmd returns the Cobra command allowing to migrate the db up to latest scheme
+func NewMigrateDBCmd(parseCfg *parse.Config) *cobra.Command {
+	return &cobra.Command{
+		Use:     "migrate-db",
+		Short:   "Migrates database to latest scheme from database/scheme folder",
+		Example: "bdjuno migrate-db",
```

Comment:
I would suggest changing the name of the command to `database migrate`

Reply:
Fixed